packages/subgraph/scripts/import-plugin-repo.ts

```diff
@@ -1,29 +1,45 @@
+import {SupportedNetworks} from '@aragon/osx-commons-configs';
 import dotenv from 'dotenv';
 import fs from 'fs';
 import path from 'path';
 
+const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
```

Comment:
We have address zero in the commons sdk (https://github.com/aragon/osx-commons/blob/develop/sdk/src/constants.ts#L41)
can't we use it?

Reply:
Discussion: Currently `sdk-commons` is not being used as a package in the subgraph and putting it as dependencies only for using a constant may not be worth it. There is also a constant  in [the tests ](https://github.com/aragon/osx-plugin-template-hardhat/blob/deaef7ada8886605706a72c2a832f3c5eda73b51/packages/subgraph/tests/utils/constants.ts#L6) but won't feel good to use it. 

New following task will be created to go over this